Includes/menu.cpp

```diff
@@ -175,6 +184,62 @@ void MenuXbe::execute(Menu *menu) {
   }
 }
 
+std::vector<std::shared_ptr<MenuItem>> *MenuXbe::getChildNodes() {
+  std::lock_guard<std::mutex> lock(childNodesLock);
+  return MenuNode::getChildNodes();
+}
+
+void MenuXbe::updateScanningLabel() {
+  std::string scanningLabel = "Scanning \"" + remainingScanPaths.front() + "\" ...";
+  {
```

Comment:
Extra braces?

Reply:
Oops; holdover from before I split this out into a separate method where I wanted to minimize the time the mutex was held. Removed.